command_server/generic/base.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+import json
+import aiohttp
+
+
+class BaseView(object):
+
+    @classmethod
+    def as_view(cls):
+        view = cls()
+        return view
+
+    @asyncio.coroutine
```

Comment:
You use decorator caroutine for dispatch method and then call one of http  method again with coroutine decorator. It seems that you should use it in one place